README.md

```diff
@@ -40,6 +40,9 @@ Components
  - modal
  - sidebar
 
+### Env variables
+ - Google API key
```

Comment:
Explain what it is what for (1 line)
Add the notion .env, all the .env shall be described here